radioactive-decay/content.md

```diff
@@ -0,0 +1,43 @@
+# <div style ="text-align: right">Radioactive Decay</div>
```

Comment:
I think you can get rid of this pull request. I initially did it as practice to get used to git, etc., and I'm not done with this node anyway. But I'll remove html.